### PR TITLE
JBPM-5291 Process Definitions: Project and Deployment Name coupled

### PR DESCRIPTION
**Pull request by bxf12315**
@cristianonicolai  Please review it.

### CODE REVIEW

**Comment by bsremac** [comment]
@bxf12315 Sorting by deployment doesn't seem to work. 
Have a look at the same list which is sorted by the old _Project_ column (correct) and the new _Deployment_ column (incorrect), in descending order.
sortingbyproject
sortingbydeployment

**Comment by bxf12315** [comment]
@bsremac  I will check it.

**Comment by bxf12315** [comment]
@bsremac  I raise another PR for it . Please check it.https://github.com/kiegroup/jbpm/pull/1261

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java [hunk 1]
```diff
@@ -166,16 +166,16 @@ public String getValue(ProcessSummary object) {
...
             @Override
             public String getValue(ProcessSummary object) {
                 return object.getDeploymentId();
             }
         };
-        projectColumn.setSortable(true);
-        projectColumn.setDataStoreName(COL_ID_PROJECT);
-        return projectColumn;
+        deploymentColumn.setSortable(true);
+        deploymentColumn.setDataStoreName(COL_ID_DEPLOYMENT);
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java]: @bxf12315 if you keep the COL_ID_PROJECT in here, you could avoid the backend change, right?

**Comment by bxf12315** [comment]
@cristianonicolai Updated it.
Please check it.

**Comment by bsremac** [comment]
@cristianonicolai @bxf12315 All is good from UI side, although must agree that introducing **Deployment** as column name in some places and keeping **Project** as column name in others, in the code itself, can lead to hard-to-read code moving forward.

**Pull request merged**